lib/workers/repository/onboarding/pr/index.ts

```diff
@@ -39,6 +41,24 @@ export async function ensureOnboardingPr(
   logger.trace({ config });
   // TODO #7154
   const existingPr = await platform.getBranchPr(config.onboardingBranch!);
+  if (existingPr) {
+    // skip pr-update if branch is conflicted
+    if (
+      await isOnboardingBranchConflicted(
+        config.defaultBranch!,
+        config.onboardingBranch!
+      )
+    ) {
+      await ensureComment({
+        number: existingPr.number,
+        topic: 'Branch Conflicted',
+        content: emojify(
+          `:warning: This PR has a merge conflict which Renovate is unable to automatically resolve, so updates to this PR description are now paused. Please resolve the merge conflict manually.\n\n`
+        ),
+      });
```

Comment:
I think this comment should be removed once the conflicts are resolved

Reply:
Or, if we'll never have conflicts in oboarding branch, maybe just remove the entire commenting logic?

Reply:
It could happen due to a bad merge  although I could live without comments if it saves us anything

Reply:
Do we really save anything by removing this comment though?

Reply:
> Do we really save anything by removing this comment though?

Please elaborate, I'm not sure what do you mean

Reply:
I see that I misunderstood your original comment. I thought you meant we shouldn't add this comment. But you meant we should remove it after it's added.

Given how rare this case is, I think we can live without removing it.

Reply:
As well as without adding it at all, though actually it isn't big concern to me